src/dev/renovate/config.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { RENOVATE_PACKAGE_GROUPS } from './package_groups';
+import { wordRegExp, maybeFlatMap, maybeMap } from './utils';
+
+const DEFAULT_LABELS = ['release_note:skip', 'renovate', 'v8.0.0', 'v7.3.0'];
```

Comment:
optional: is there a way to get current versions? as I can `package.json` contains reference to 8.0.0. it's easy to forget to update renovate when bump to the new version.

Reply:
Not that I'm aware of. I could pull in the current master version mapping, but the other version will change much more frequently. The closest thing I could find is the `.backportrc.json` file, but that just mentions the default branch names (`7.x`). Ultimately I'm not worried about having to update this after version bumps. After a while I might get sick of that, but it will be relatively infrequent.